contracts/Uni.sol

```diff
@@ -19,13 +19,8 @@ contract Uni {
     /// @notice Address which may mint new tokens
     address public minter;
 
-    /// @notice The timestamp after which minting may occur
-    uint public mintingAllowedAfter;
-
-    /// @notice Minimum time between mints
-    uint32 public constant minimumTimeBetweenMints = 1 days * 365;
-
     /// @notice Cap on the percentage of totalSupply that can be minted at each mint
+    /// TODO: Need to think and decide on whether to keep this or not  
     uint8 public constant mintCap = 2;
```

Comment:
We need to decide if we still want this

Reply:
We should remove the mintCap from core contract, this can be introduced through a `minter` contract.

Reply:
Okay so should we write a `minter` contract at this stage? What other functionalities do we need in this `minter`?

Reply:
Also, done removing the mintCap